src/hermes/commands/base.py

```diff
@@ -131,10 +131,13 @@ def init_command_parser(self, command_parser: argparse.ArgumentParser) -> None:
 
     def load_settings(self, args: argparse.Namespace):
         """Load settings from the configuration file (passed in from command line)."""
-
-        toml_data = toml.load(args.path / args.config)
-        self.root_settings = HermesCommand.settings_class.model_validate(toml_data)
-        self.settings = getattr(self.root_settings, self.command_name)
+        try:
+            toml_data = toml.load(args.path / args.config)
+            self.root_settings = HermesCommand.settings_class.model_validate(toml_data)
+            self.settings = getattr(self.root_settings, self.command_name)
+        except FileNotFoundError as e:
+            self.log.error("hermes.toml was not found.")
```

Comment:
How about some helpful advice?

```suggestion
            self.log.error("hermes.toml was not found. Try to run 'hermes init' first or create one manually.")
```